dashboard/of-cloud-dashboard/handler.js

```diff
@@ -207,7 +212,9 @@ var isResourceInTokenClaims = function (path, queryString, user, organisations)
   if (user === queryString["user"]) {
     return true
   }
-  return organisations.indexOf(queryString["user"]) >= 0
+
+  let orgs = organisations.split(",");
```

Comment:
Can `organisations` be undefined? What if it was zero lengthed?

Reply:
No it can't be undef, if we dont have a cookie we return above, otherwise its an empty string.

and "".split(",") --> [""], so no error

Reply:
No it can't be undef, if we dont have a cookie we return above, otherwise its an empty string.

and "".split(",") --> [""], so no error